FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -1684,7 +1684,14 @@ extension Auth: AuthInterop {
           // UIApplication responds to it.
           return
         }
-        let application = UIApplication.shared
+
```

Comment:
Nit: please add a comment mentioning why this is the case. Also delete the above check to GoogleUtilities since that doesn't work anymore.

Reply:
I'll add the comment. 

Shouldn't the GoogleUtilities function still work?  It seems safer to keep it, in case some extensions still respond to sharedApplication?

Reply:
That's fair, sgtm